src/validators/server.js

```diff
@@ -4,7 +4,15 @@ import { CLIENTS } from '../db';
 
 function serverAddressValidator(ctx) {
   const { host, port, socketPath } = ctx.obj;
-  if ((!host && !port && !socketPath) || ((host || port) && socketPath)) {
+
+  if (!host && !socketPath) {
```

Comment:
Since these two if statements are returning the same result could you please combine nem into a single if statement checking?

Reply:
I left them separate for clarity, but on second look I agree with you. Done